src/panel/poi/PoiBlockContainer.jsx

```diff
@@ -29,18 +33,38 @@ export default class PoiBlockContainer extends React.Component {
     const recyclingBlock = blocks.find(b => b.type === 'recycling');
     const covidBlock = blocks.find(b => b.type === 'covid19');
     const displayCovidInfo = this.props.covid19Enabled && blocks.find(b => b.type === 'covid19');
+    const wikipedia = informationBlock
+      ? informationBlock.blocks.find(b => b.type === 'wikipedia')
+      : null;
 
     return <div className="poi_panel__info">
+      {wikipedia && <WikiBlock block={wikipedia} />}
       {displayCovidInfo &&
-        <CovidBlock block={covidBlock} countryCode={this.props.poi.address.country_code} />
+        <>
+          <CovidBlock block={covidBlock} countryCode={this.props.poi.address.country_code} />
+          <Divider />
+        </>
       }
-      {hourBlock && <HourBlock block={hourBlock} covid19enabled={!!displayCovidInfo} />}
+      {websiteBlock && <WebsiteBlock block={websiteBlock} poi={this.props.poi} />}
       {informationBlock && <InformationBlock block={informationBlock} />}
       {phoneBlock && <PhoneBlock block={phoneBlock} />}
+      {hourBlock && <HourBlock block={hourBlock} covid19enabled={!!displayCovidInfo} />}
       {recyclingBlock && <RecyclingBlock block={recyclingBlock} />}
-      {websiteBlock && <WebsiteBlock block={websiteBlock} poi={this.props.poi} />}
       {contactBlock && <ContactBlock block={contactBlock} />}
-      {imagesBlock && <ImagesBlock block={imagesBlock} poi={this.props.poi} />}
+      {this.props.poi.address.label &&
+        <Block
+          icon="map-pin"
+          title={_('Address')}
```

Comment:
You should use the `'address'` string, without the first cap, and capitalize using `u-firstCap` or `capitalizeFirst`.

Reply:
Ok, I switched all block titles to lowercase